docs/source/slots/anatomy.md

```diff
@@ -0,0 +1,58 @@
+# Slots anatomy
+
+The slots are insertion points in the Volto rendering tree structure. You can add a
+component, along with its configuration, if any and it will be rendered in that
+insertion point. You can control in which route do you want that element appear as well
+as the order (position) of the items that you add in the in the slots. Slots are named,
+so you can add in the configuration object:
+
+```js
+export const slots = {
+  aboveContentTitle: [
+    // List of components (might have config too, maybe in `data` property)
+    { path: '/', component: 'Component', data: {} },
+    // It can include blocks too (makes sense when we will be able to save them)
```

Comment:
It's an initial thought about the future of slots, first iteration can have only "viewlet-ish", no sved config. In a second iteration we can provide a full backend support for saving those persistently, and have the ability to save slots state so we can even use blocks inside slots.